cdbot/cogs/cyber.py

```diff
@@ -346,6 +347,18 @@ def __init__(self, bot: Bot):
 
         await self.countdown('5th March 2019', 'CyberStart Essentials', ctx)
 
+    @command()
+    async def hundered(self, ctx: Context):
+        """
+        Gets the number of 100% and true 100% users
+        """
+
+        game_r = utils.get(ctx.guild.roles, id=HUNDERED_ROLE_ID)
+        true_r = utils.get(ctx.guild.roles, id=TRUE_HUNDERED_ROLE_ID)
+
+        await ctx.send((f"There are {len(game_r.members)} 100% users, of which {len(true_r.members)} have also "
```

Comment:
Maybe this could be more specific?
```suggestion
        await ctx.send((f"There are {len(game_r.members)} users who completed game, of which {len(true_r.members)} have also "
```